uploads/submission/student_1/assignment_1/2017-09-23-140332_submission/a01.py

```diff
@@ -0,0 +1,10 @@
+def code_holder(num1, num2):
```

Comment:
As before, add individual files with `git add` so that we don't get all this extra stuff in the commit.